iroha/src/config.rs

```diff
@@ -92,6 +92,15 @@ impl Configuration {
         Ok(())
     }
 
+    /// Load trusted peers variables from a json *pretty* formatted file.
+    pub fn load_trusted_peers_from_path<P: AsRef<Path> + Debug>(
+        &mut self,
+        path: P,
+    ) -> Result<(), String> {
+        self.sumeragi_configuration.trusted_peers = TrustedPeers::from_path(&path)?;
```

Comment:
May be `Configuration::from_path` can take 2 arguments one of which can be path of config and other path of trusted peers?